client/protocols/unlock.py

```diff
@@ -7,13 +7,27 @@
 
 from client.protocols.common import models
 from client.utils import format
-from client.utils import locking
+from client.utils import locking, auth
 from datetime import datetime
 import logging
 import random
 import string
+from urllib.request import urlopen
+import json
+import os
+import sys
 
 log = logging.getLogger(__name__)
+countDir = "countDir"
+TGSERVER = "http://127.0.0.1:5000/"
+
+try:
+    with open("tests/.ok_guidance","r") as f:
```

Comment:
Don't put this code at top level. It would be better to put it in UnlockProtocol.__init__ (or in a method called from __init__) and make `json_dict_file` and `error` attributes instead of global variables.